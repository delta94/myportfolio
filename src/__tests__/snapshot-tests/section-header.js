import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import SectionHeader from '../../components/section-header';

it('SectionHeader renders correctly', () => {
  const wrapper = shallow(<SectionHeader title="Prabhu Murthy Portfolio" color="#ddd" />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
