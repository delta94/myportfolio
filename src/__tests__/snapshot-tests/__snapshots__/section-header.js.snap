// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader renders correctly 1`] = `
<styled.header
  color="#ddd"
>
  Prabhu Murthy Portfolio
</styled.header>
`;
